astropy/table/jsviewer.py

```diff
@@ -168,23 +158,23 @@ def html_js(self, table_id='table0'):
                                      tid=table_id).strip()
 
 
-def write_table_jsviewer(table, filename, table_id=None,
-                         css="table,th,td,tr,tbody {border: 1px solid black; border-collapse: collapse;}",
-                         max_lines=5000,
-                         jskwargs={}):
+def write_table_jsviewer(table, filename, table_id=None, max_lines=5000,
```

Comment:
Indeed, this seems to go back to https://github.com/astropy/astropy/commit/d18364dd749f3250f5a16f0eb45c45e344988a9e when the use of pformat has been replaced with the HTML write. Fixed with table slicing.